src/ext-test/java/org/opentripplanner/ext/fares/impl/OrcaFareServiceTest.java

```diff
@@ -340,30 +341,45 @@ void calculateFareForWSFPtToTahlequah() {
    * Single trip with Link Light Rail to ensure distance fare is calculated correctly.
    */
   @Test
-  void calculateFareForLightRailLeg() {
+  void calculateFareForSTRail() {
     List<Leg> rides = List.of(
-      getLeg(SOUND_TRANSIT_AGENCY_ID, "1-Line", 0, "Roosevelt Station", "Int'l Dist/Chinatown")
+      getLeg(SOUND_TRANSIT_AGENCY_ID, "1-Line", 0, "Roosevelt Station", "Int'l Dist/Chinatown"),
+      getLeg(SOUND_TRANSIT_AGENCY_ID, "S Line", 100, "King Street Station", "Auburn Station")
     );
-    calculateFare(rides, regular, DEFAULT_TEST_RIDE_PRICE);
-    calculateFare(rides, FareType.senior, DEFAULT_TEST_RIDE_PRICE);
+    calculateFare(rides, regular, DEFAULT_TEST_RIDE_PRICE.times(2));
+    calculateFare(rides, FareType.senior, DEFAULT_TEST_RIDE_PRICE.times(2));
     calculateFare(rides, FareType.youth, Money.ZERO_USD);
     calculateFare(rides, FareType.electronicSpecial, ORCA_SPECIAL_FARE);
     calculateFare(rides, FareType.electronicRegular, DEFAULT_TEST_RIDE_PRICE);
     calculateFare(rides, FareType.electronicSenior, ONE_DOLLAR);
     calculateFare(rides, FareType.electronicYouth, Money.ZERO_USD);
   }
 
+  /**
+   * Test King County Water Taxis
+   */
   @Test
-  void calculateFareForSounderLeg() {
+  void calculateWaterTaxiFares() {
     List<Leg> rides = List.of(
-      getLeg(SOUND_TRANSIT_AGENCY_ID, "S Line", 0, "King Street Station", "Auburn Station")
+      getLeg(KC_METRO_AGENCY_ID, "973", 1)
     );
-    calculateFare(rides, regular, DEFAULT_TEST_RIDE_PRICE);
-    calculateFare(rides, FareType.senior, DEFAULT_TEST_RIDE_PRICE);
+    calculateFare(rides, regular, usDollars(5.75f));
```

Comment:
Can you extract constants for the amounts that you're re-using? IntelliJ's "Extract constant" feature will make this quite easy and replace all the other instances for you.